pkgs/development/libraries/glibc/default.nix

```diff
@@ -14,16 +14,19 @@ let
   ];
 in
 
-callPackage ./common.nix { inherit stdenv; } {
-    pname = "glibc" + lib.optionalString withGd "-gd";
+(lib.makeOverridable (callPackage ./common.nix { inherit stdenv; }) {
+  inherit withLinuxHeaders withGd;
+  pname = "glibc" + lib.optionalString withGd "-gd";
+}).overrideAttrs(previousAttrs: {
```

Comment:
```patch
diff --git a/pkgs/development/libraries/glibc/default.nix b/pkgs/development/libraries/glibc/default.nix
index 6fc735f83c2..8cee86162a0 100644
--- a/pkgs/development/libraries/glibc/default.nix
+++ b/pkgs/development/libraries/glibc/default.nix
@@ -14,13 +14,10 @@ let
   ];
 in
 
-(lib.makeOverridable (callPackage ./common.nix { inherit stdenv; }) {
-  inherit withLinuxHeaders withGd;
+((callPackage ./common.nix { inherit stdenv; }) {
+  inherit withLinuxHeaders withGd profilingLibraries withLibcrypt;
   pname = "glibc" + lib.optionalString withGd "-gd";
 }).overrideAttrs(previousAttrs: {
-
-    inherit profilingLibraries withLibcrypt;
-
     # Note:
     # Things you write here override, and do not add to,
     # the values in `common.nix`.
```

`profilingLibraries withLibcrypt` are args

makeOverridable doesn't seem necessary

Reply:
Thanks, I can never seem to figure out when it is or isn't required.

Reply:
How did you get github to display a nice pretty diff like that?

Reply:
> How did you get github to display a nice pretty diff like that?

made the edits on my pc and then used
\`\`\`patch
text
\`\`\`

Reply:
>  ```patch

Ah, that is the trick I was looking for.  Thank you!  I'm writing a TUI github client (since I have abandoned any hope of nixpkgs leaving github) and this is really useful.